.github/workflows/releases.yaml

```diff
@@ -0,0 +1,69 @@
+name: Build and Publish Release Binaries
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        goos: [linux, windows, darwin]
```

Comment:
Based on the discussion [here](https://github.com/ethereum/go-ethereum/issues/26170#issuecomment-1318467453) I'm not sure windows will work. We can try and see.